Makefile

```diff
@@ -40,4 +40,3 @@ format:
 	clang-format --sort-includes=0 -style=file -i postgres_scanner.cpp
 	clang-format --sort-includes=0 -style=file -i concurrency_test.cpp
```

Comment:
Perhaps we should include the same `.clang-format` file as used in the main repo?

Reply:
It probably should you're right